go.mod

```diff
@@ -1,10 +1,13 @@
 module github.com/ipfs/go-datastore
 
 require (
+	github.com/dvyukov/go-fuzz v0.0.0-20200318091601-be3528f3a813 // indirect
 	github.com/google/uuid v1.1.1
+	github.com/ipfs/go-ds-badger v0.2.1
+	github.com/ipfs/go-ds-leveldb v0.4.2
```

Comment:
do we want to add these as hard deps? Perhaps the fuzz portion needs to become a sub-project? @Stebalien thoughts?

Reply:
We don't. We should create a sub-module.